html/feed/feed.js

```diff
@@ -1,5 +1,369 @@
+iLike.className = 'fas fa-heart text-danger'; 
+btnLike.appendChild (iLike); 
+
+const spanLike = document.createElement ('span'); 
+spanLike.id= `likeCount${element.id}`; 
+spanLike.textContent = element.reactions [0]?.count; 
+
+console.log (element)
+btnLike.appendChild (spanLike); 
+
+// edit btn 
+
+if (userName === element.author.name){
+
+  const btnEdit = document.createElement ('button'); 
+  btnEdit.className = 'btn btn-link btn-sm text-muted'; 
+  
+  const iEdit = document.createElement ('i'); 
+  iEdit.className = 'fas fa-edit'; 
+  btnEdit.appendChild (iEdit); 
+
+  const spanEdit = document.createElement ('span'); 
+  spanEdit.textContent = "Edit"; 
+  btnEdit.setAttribute ('data-post-id', element.id); 
+  confirmEditBtn.setAttribute ('data-post-id', element.id); 
+  btnEdit.appendChild (spanEdit)
+
+  //add eventListener to edit post
+
+  btnEdit.addEventListener ('click',function (){
+
+    document.getElementById ('editTitle').value = element.title; 
+    document.getElementById ('editBody').value = element.body; 
+
+    var modal =  new bootstrap.Modal (getElementById ('editPostModal')); 
+
+    modal.show ();
+  }); 
+
+  btnGroup.appendChild (btnEdit);
+}
+
+// likeBtn eventListener 
+
+btnLike.addEventListener ("click", (e)=>{
+  reactToPosts (baseUrl, element.id, token, spanlike)
+})
+
+btnGroup.appendChild (btnComment); 
+btnGroup.appendChild (btnLike); 
+
+cardBody.appendChild (btnGroup); 
+
+colContent.appendChild (cardBody); 
+
+row.appendChild (colImage); 
+row.appendChild (colContent); 
+
+card.appendChild (row); 
+
+// commentSection 
+
+const commentCollapse = document.createElement ('div'); 
+commentCollapse.className = 'collapse mt-3'; 
+commentCollapse.id = `commentCollapse${element.id}`; 
+
+// create the form 
+
+const commentForm = document.createElement ('form'); 
+
+const commentInput = document.createElement ('input'); 
+commentInput.className = 'form-control'; 
+commentInput.type = 'text'; 
+commentInput.placeholder = 'Write a reply...'; 
+commentForm.appendChild (commentInput); // append to form insted 
+
+// add a hidden submit btn- allows users to press enter to submit. 
+
+const hiddenSubmitBtn = document.createElement ('button'); 
+hiddenSubmitBtn.style.display = 'none'; 
+commentForm.appendChild (hiddenSubmitBtn); 
+
+commentForm.addEventListener ("submit", (e)=> {
+  e.preventDefault ()
+    const commentText = commentInput.value; 
+    if (commentText){
+      postComment (baseUrl, element.id, commentText, token, spancomment, element); 
+      commentInput.value = ''; 
+
+    }
+
+})
+
+commentCollapse.appendChild (commentForm); 
+
+//modify btnComment to controll the collapse of comment section 
+
+btnComment.setAttribute ('data-bs-toggle','collapse'); 
+btnComment.setAttribute ('data-bs-target','#commentCollapse${element.id}'); 
+btnComment.setAttribute ('aria-expanded','false'); 
+btnComment.setAttribute ('aria-controls','commentCollapse${element.id}'); 
+    
```

Comment:
please use correct quotations when using javascript variables inside a "" you need to use correct backtics or else the variable will only be read as a string, it wont open the ability to comment because of this.

Reply:
change quotations. replaced with backticks in the necessary places.